packages/playwright/src/runner/testServer.ts

```diff
@@ -387,6 +388,11 @@ export class TestServerDispatcher implements TestServerInterface {
     if (process.env.PWTEST_DEBUG)
       return;
     if (intercept) {
+      // Don't intercept any DEBUG=* logging
+      debug.log = (...args) => {
```

Comment:
- Let's not overwrite when `debug.log` is already set? We set it to a different value in `debugLogger.ts`.
- We should also revert back like we do with `stdout/stderr`.